handlers/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+- name: drush clear cache
```

Comment:
Maybe this should be called `drush clear drush cache`, or even `drush cache clear drush` to avoid any ambiguity in terms of which caches are being cleared by the handler?

I wouldn't want users to try adding `drush clear cache` to their playbook somewhere, thinking it's actually clearing _Drupal_'s cache...